Example/ios/Example/AppDelegate.m

```diff
@@ -31,7 +31,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
    * on the same Wi-Fi network.
    */
 
-  jsCodeLocation = [NSURL URLWithString:@"http://192.168.179.57:8081/index.ios.bundle?platform=ios&dev=true"];
+  jsCodeLocation = [NSURL URLWithString:@"http://192.168.50.35:8081/index.ios.bundle?platform=ios&dev=true"];
```

Comment:
Sure, just changed it for local testing and never set it back 😑 
As updating React Native makes sense, this shouldn´t be an issue for long.